generic-examples/languages/README.md

```diff
@@ -1,3 +1,23 @@
-# Supported languages Camel K examples
+# Supported languages for writing Camel K integrations
 
-Find useful examples about how to develop a Camel K integration in `Java`, `Javascript`, `Groovy`, `XML` and `YAML`.
\ No newline at end of file
+Contains useful examples on how to develop a Camel K integration in `Java`, `Javascript`, `Groovy`, `XML`, `Kotlin` and `YAML`.
```

Comment:
A bit nitpicky but it should be `JavaScript`

Reply:
Done!